packages/DApp/package.json

```diff
@@ -17,6 +17,7 @@
     "clean": "rimraf node_modules"
   },
   "dependencies": {
+    "@status-im/js": "*",
```

Comment:
As dep, otherwise copy over

- protobuf dependencies and definition
- key deserialization fn
- key recovery fn
- id creation fn
- more